client/cmd/internal/plan/plan_test.go

```diff
@@ -152,6 +153,57 @@ func TestPlanGetResult(t *testing.T) {
 		assertMapPlanMatch(t, actual.Job.Migrate, expected.Job.Migrate)
 	})
 
+	t.Run("return empty plan when create and delete job on same namespace", func(t *testing.T) {
```

Comment:
should it be return plan with update if create and delete job on same namespace?

Reply:
we need to make sure, if there are any updates from spec side mas @deryrahman . Its same context why apply command we have namespace migrate and update operation.

Reply:
got it